azure-pipelines.yml

```diff
@@ -68,15 +68,20 @@ jobs:
     variables:
       VSINSTALLDIR: C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\
       HOST_DMD_VERSION: 2.093.1
+      D_COMPILER: ldc
+      VISUALD_VER: v0.49.0
+      LDC_VERSION: 1.20.0
     strategy:
       matrix:
         win32-ldc:
           OS: Win_32
-          MODEL: 32
+          MODEL: 32mscoff
```

Comment:
This is no real change, `MODEL` is overwritten in https://github.com/dlang/dmd/blob/3a55c54a8e5773c3f56e10365a98f4570f8c6ce6/.azure-pipelines/windows-msbuild.bat#L9, so this is just for clarity (testing MSCOFF, not OMF).